src/styles/_catalog.scss

```diff
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition-duration: 300ms;
+    transition-property: all;
```

Comment:
Always add transition property. All is default value, and it's hard for the browser to control all properties